conanfile.py

```diff
@@ -0,0 +1,435 @@
+        "hdri": [True, False],
+        "magickpp": [True, False],
+        "quantum_depth": [8, 16, 32],
+        "utilities": [True, False],
+        "with_bzlib": [True, False],
+        "with_cairo": [True, False],
+        "with_djvu": [True, False],
+        "with_fftw": [True, False],
+        "with_fontconfig": [True, False],
+        "with_freetype": [True, False],
+        "with_heic": [True, False],
+        "with_jbig": [True, False],
+        "with_jpeg": [None, "libjpeg", "libjpeg-turbo"],
+        "with_jxl": [True, False],
+        "with_lcms": [True, False],
+        "with_lzma": [True, False],
+        "with_opencl": [True, False],
+        "with_openexr": [True, False],
+        "with_openjp2": [True, False],
+        "with_openmp": [True, False],
+        "with_pango": [True, False],
+        "with_png": [True, False],
+        "with_raw": [True, False],
+        "with_threads": [True, False],
+        "with_tiff": [True, False],
+        "with_webp": [True, False],
+        "with_x11": [True, False],
+        "with_xml2": [True, False],
+        "with_zlib": [True, False],
+        "with_zstd": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "cipher_support": True,
+        "exclude_deprecated": True,
+        "hdri": True,
+        "magickpp": True,
+        "quantum_depth": 16,
+        "utilities": True,
+        "with_bzlib": True,
+        "with_cairo": True,
+        "with_djvu": False,
+        "with_fftw": True,
+        "with_fontconfig": True,
+        "with_freetype": True,
+        "with_heic": True,
+        "with_jbig": True,
+        "with_jpeg": "libjpeg",
+        "with_jxl": False,  # FIXME: re-enable once migrated
+        "with_lcms": True,
+        "with_lzma": True,
+        "with_opencl": True,
+        "with_openexr": True,
+        "with_openjp2": True,
+        "with_openmp": True,
+        "with_pango": False,  # FIXME: re-enable once migrated
+        "with_png": True,
+        "with_raw": True,
+        "with_threads": True,
+        "with_tiff": True,
+        "with_webp": True,
+        "with_x11": True,
+        "with_xml2": True,
+        "with_zlib": True,
+        "with_zstd": True,
+    }
+    exports_sources = [
+        "CMakeLists.txt",
+        "configure",
+        "LICENSE",
+        "NOTICE",
+        "cmake/*",
+        "coders/**",
+        "config/**",
+        "filters/**",
+        "images/**",
+        "m4/**",
+        "Magick++/**",
+        "MagickCore/**",
+        "MagickWand/**",
+        "utilities/**",
+    ]
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        if self.settings.os not in ["Linux", "FreeBSD"]:
+            del self.options.with_x11
+        if is_msvc(self):
+            # TODO: fix MSVC compilation with these options
+            self.options.with_fftw = False
+            self.options.with_openmp = False
```

Comment:
What was the issue here? I use it with openmp on windows currently.

Reply:
It failed to link. I got `__imp_*` openmp symbol errors. FFTW one was confusing as the error happens inside a large macro.